src/commands/gate/__tests__/evaluate.test.ts

```diff
@@ -116,6 +116,22 @@ describe('evaluate', () => {
       expect(stdErrLog).not.toContain('internal issue')
     })
   })
+  describe('wait', () => {
+    test('should pass the command if the response status is wait', () => {
+      const write = jest.fn()
+      const command = new GateEvaluateCommand()
+      command.context = {stdout: {write}} as any
```

Comment:
You can now refactor this. It will take care of resolving the default values for your command options.
```suggestion
      const command = createCommand(GateEvaluateCommand, {stderr: {write}} as any)
```